cmake/Modules/CodingStandard.cmake

```diff
@@ -0,0 +1,21 @@
+if(CMAKE_VERSION VERSION_LESS 3.12)
+    find_package(PythonInterp) # Deprecated since version 3.12
```

Comment:
This should be looking for Python 3. So something more similar to https://github.com/lammps/lammps/blob/ec6d3586ce30a4fd62562272b264c6ad9453c567/cmake/Modules/Documentation.cmake